packages/editor/CodeMirror/theme.ts

```diff
@@ -182,7 +182,7 @@ const createTheme = (theme: EditorTheme): Extension[] => {
 
 		// Override the default URL style when the URL is within a link
 		'& .tok-url.tok-link, & .tok-link.tok-meta, & .tok-link.tok-string': {
-			opacity: theme.isDesktop ? 0.6 : 1,
+			opacity: 0.6,
```

Comment:
This change causes a contrast-related accessibility issue with the default theme:
![screenshot: Contrast checker: #00337799 with opacity 0.6 has contrast 3.8:1](https://private-user-images.githubusercontent.com/46334387/325080264-270b5792-c22f-4245-b57c-d273dba9dbac.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTM5MzMzNDksIm5iZiI6MTcxMzkzMzA0OSwicGF0aCI6Ii80NjMzNDM4Ny8zMjUwODAyNjQtMjcwYjU3OTItYzIyZi00MjQ1LWI1N2MtZDI3M2RiYTlkYmFjLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA0MjQlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNDI0VDA0MzA0OVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTczZmU4Y2JhMzcxOGMxZmI2ZGJiMDdhNTQ5ZTNlNzhiZjYxYTE2NTgxMTNkZTU5YzQwZmVhNmVmZTk1NWNkZGImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.cywXqBtMehW1L5bivEHHec7QPPilO7pFAvo9MsGEbEA)